client/js/components/procedure/basicSettings/DpBasicSettings.vue

```diff
@@ -168,8 +229,31 @@ export default {
       this.selectedPublicPhase = phase
     },
 
+    submit () {
+      const addonExists = Boolean(window.dplan.loadedAddons['addon.additional.field'])
+      const addonHasValue = this.addonPayload.value || this.addonPayload.initValue
```

Comment:
Don't we want this to be of type Boolean, too?

Reply:
I find it more precise and probably less error prone. Done here https://github.com/demos-europe/demosplan-core/pull/3999/commits/05b4d0a61fe1af1d235ee5290657a05239eec642